framework/base/ErrorException.php

```diff
@@ -17,6 +17,8 @@
  */
 class ErrorException extends \ErrorException
 {
+    const E_HHVM_FATAL_ERROR = 16777217;
```

Comment:
16777217 comes from here: https://github.com/facebook/hhvm/blob/master/hphp/runtime/base/runtime-error.h#L62 More info is in this issue https://github.com/facebook/hhvm/issues/4818 in comment from iansltx.